ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/altus/GrpcMinaSshdManagementClient.java

```diff
@@ -228,7 +229,7 @@ public MinaSshdService acquireMinaSshdServiceAndWaitUntilReady(String requestId,
             Supplier<CcmException> timeoutExceptionSupplier = () -> new CcmException(String.format("Timed out while trying to %s", actionDescription), true);
 
             return RetryUtil.performWithRetries(
-                    () -> client.unregisterSshTunnelingKey(requestId, DUMMY_MINASSHD_SERVICE_ID, keyId), actionDescription,
+                    () -> client.unregisterSshTunnelingKey(requestId, minaSshdServiceId, keyId), actionDescription,
```

Comment:
DUMMY_MINASSHD_SERVICE_ID can be removed

Reply:
thanks, removed